lib/pages/chat_list/chat_list.dart

```diff
@@ -593,11 +516,7 @@ class ChatListController extends State<ChatList>
   }
 
   void cancelAction() {
-    if (selectMode == SelectMode.share) {
-      setState(() => Matrix.of(context).shareContent = null);
-    } else {
-      setState(() => selectedRoomIds.clear());
-    }
+    setState(() => selectedRoomIds.clear());
   }
```

Comment:
IMO: Change `stateState` to `ValueNotifier`

Reply:
not belong to ticket, later